<launch>
    <param name="keyboard_topic" value = "/keyboard_input"/>
    <param name="speech_topic" value = "/speech_recog"/>
    <param name = "goal_topic"  value = "/user_goal"/>
    <node name="speech_recog" pkg="logikoma_multi_modal" type = "SpeechRecog.py" />
    <node name="logikoma_user_input" pkg = "logikoma_multi_modal" type = "Logikoma_User_Input.py">
        <param name="use_keyboard" type="bool" value = "true"/>
        <param name="use_speech" type="bool" value="true"/>
    </node>
</launch>